### ISSUE TITLE
Create a multi-stage container for golang api

### ISSUE THREAD

**Issue by jstoja** [felix-fabrega/etix-tv-manager#36]
I think you can use [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/#use-multi-stage-builds) with Docker in order to use a smaller image and still have 1 dockerfile (I saw that you commented out the "future" file).

**Comment by felix-fabrega** [comment]
Yes definitely. I plan to re-work the whole building and running stage to have a proper dev and prod environment. For now I had to rush everything for my POC deadline, should be better after some rework :p 
Thanks for the suggestion I will use this way to build lighter container.
